### DISCUSSION TITLE
Welcome to allchat Discussions!

### DISCUSSION THREAD

**Discussion by msveshnikov** [msveshnikov/allchat#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Cristofer543** [comment]
Good

**Comment by jsugg** [comment]
Nice project, loved it!
